addons/expanded-backpack/addon.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "Expanded backpack",
+  "description": "Adds multiple rows, better previews and vertical scrolling to the editor backpack to make it easier to find things.",
+  "credits": [
+    {
+      "name": "pumpkinhasapatch",
+      "link": "https://github.com/pumpkinhasapatch"
+    }
+  ],
+  "versionAdded": "1.27.3",
+  "tags": ["editor"],
```

Comment:
"theme" tag?

Reply:
The purpose of this addon is to improve usability of the backpack, not the appearance.